src/ugeneui/html/version_news_en.html

```diff
@@ -18,14 +18,34 @@
  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
  MA 02110-1301, USA.
 -->
-<h3>Changes in UGENE v. 39</h3>
+<h3>Changes in UGENE v. 40</h3>
+
+<h4>Alignment Editor</h4>
+<ul>
+    <li style="margin-bottom: 10px;"> Multi-row selection support.</li>
+    <li style="margin-bottom: 10px;">Built Tree dialog now has all PhyML algorithms listed.</li>
+    <li style="margin-bottom: 10px;">A new option to move selected sequences from one document to another.</li>
+    <li style="margin-bottom: 10px;">Grouping mode: sorting by the size of a group and showing group sizes in the name list.</li>
```

Comment:
**The grouping mode**: sorting by the size of a group and showing group sizes in the name list.

Reply:
Your suggestion is valid. Fixed. 
But my variant is also good and sounds more like a newspaper heading where articles usually omitted.